skt/runner.py

```diff
@@ -98,6 +122,15 @@ def dumpjunitresults(self, jobid, junit):
             (stdout, stderr) = bkr.communicate()
 
     def getconsolelog(self, jobid=None):
+        """
+        Retrieve console log from a Beaker job.
+
+        Args:
+            jobid: The Beaker jobid
```

Comment:
What is called "jobid" in the code, in [official, modern Jenkins terminology](https://jenkins.io/doc/book/glossary/) is "build ID". So, we need to describe this properly, and have a separate commit fixing the names (might be done later).

Reply:
Agreed.

Reply:
Could you please make this say "The Beaker build ID"? Sorry, if I wasn't clear before. We can change the variable name to match later. This is needed to avoid confusion between "projects" and "builds", as earlier they were both often called "job".

Reply:
Heey, I was properly tired here :) I confused Beaker and Jenkins, niiice :) This is good to go, and I just pushed this to master. Thank you, Major! I fixed it up a little bit, though: spelled out "Beaker job ID" instead of "Beaker jobid", and terminated the argument description with a full stop.